### PR TITLE
Feat/362467  sort cdos

### PR DESCRIPTION
**Pull request by whitewaterdesign**

### CODE REVIEW

File: app/api/ddi-index-api/cdos.js [hunk 1]
```diff
@@ -137,16 +137,32 @@ const getLiveCdosWithinMonth = async (sort = {}) => {
...
-const getInterimExemptions = async (sort = {}) => {
+const getInterimExemptions = async (interimExemptForSort = {}) => {
+  /**
+   * @type {Partial<CdoSort>}
+   */
+  let sortOptions = {}
+
+  if (interimExemptForSort.column !== 'interimExemptFor' && interimExemptForSort.column !== undefined) {
+    sortOptions = interimExemptForSort
+  } else if (interimExemptForSort.order === 'ASC') {
```
Comment (jbarnsley10) [inline app/api/ddi-index-api/cdos.js:156]: Does DESC also need inverting here if column is 'interimExemptFor'? Or is that covered elsewhere?

File: app/models/cdo/manage/live.js [hunk 2]
```diff
@@ -47,15 +47,17 @@ const columnLink = (tab, sort, column) => {
   let calculatedColumn = column
 
   if (column === undefined) {
-    calculatedColumn = tab === 'interim' ? 'joinedExemptionScheme' : 'cdoExpiry'
+    calculatedColumn = tab === 'interim' ? 'interimExemptFor' : 'cdoExpiry'
   }
 
-  if (calculatedColumn !== 'cdoExpiry' && calculatedColumn !== 'joinedExemptionScheme') {
+  const defaultSortOrder = calculatedColumn === 'interimExemptFor' ? ['DESC', 'ASC'] : ['ASC', 'DESC']
```
Comment (jbarnsley10) [inline app/models/cdo/manage/live.js:53]: I like this

File: app/api/ddi-index-api/cdos.js (hunk 1 above)
Comment (whitewaterdesign) [reply app/api/ddi-index-api/cdos.js:156]: DESC will use default which is the same as ASC

**Review by jbarnsley10 — APPROVED** [review]
LGTM

**Pull request merged**